### PR TITLE
Put dog fixed

### PR DESCRIPTION
**Pull request by haeramkeem**
hot dog final

### CODE REVIEW

File: src/main/java/org/cnu/realcoding/controller/DogController.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
+import java.util.Map;
```
Comment (ywnam-stephen) [inline src/main/java/org/cnu/realcoding/controller/DogController.java]: Is this necessary to import?

**Review by ywnam-stephen — COMMENTED** [review]
just nitpicking

File: src/main/java/org/cnu/realcoding/controller/DogController.java (hunk 1 above)
Comment (haeramkeem) [reply src/main/java/org/cnu/realcoding/controller/DogController.java]: doooone

Comment (ywnam-stephen) [reply src/main/java/org/cnu/realcoding/controller/DogController.java]: good boy

**Pull request merged**